pages/expenses.js

```diff
@@ -356,11 +356,11 @@ const expensesPageQuery = gqlV2/* GraphQL */ `
           name
           slug
           type
-          supportedPayoutMethods
           settings
           plan {
             id
           }
+          ...ExpenseHostFields
```

Comment:
If you're adding this, remove the fields that are duplicates.

Reply:
Thanks for the review. Done. 👍🏽